src/docker/entrypoint.sh

```diff
@@ -20,7 +20,10 @@ echo "Log level (LOG_LEVEL): $LOG_LEVEL"
 echo "Graphistry user (GRAPHISTRY_USERNAME): $GRAPHISTRY_USERNAME"
 echo ""
 
-
+if [ -n "$ADDITIONAL_PYTHON_PATH" ]; then
+    export PYTHONPATH=$PYTHONPATH:$ADDITIONAL_PYTHON_PATH
+    echo "Using ADDITIONAL_PYTHON_PATH=$ADDITIONAL_PYTHON_PATH"
```

Comment:
I think our imports should go first, not last, right?

Reply:
I see, done!